mrp_multi_level/views/mrp_area_views.xml

```diff
@@ -35,7 +35,11 @@
                     <group colspan="4" col="2">
                         <group>
                             <field name="active" invisible="1" />
-                            <field name="warehouse_id" />
+                            <field
+                                name="warehouse_id"
+                                options="{'no_create': True, 'no_create_edit': True}"
+                                force_save="1"
```

Comment:
why the force_save?

Reply:
You are right, the only change needed is `'no_create': True`